src/VisualStudio/VisualBasic/Impl/LanguageService/VisualBasicPackage.vb

```diff
@@ -73,7 +74,10 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic
                 isMainThreadTask:=False,
                 task:=Function() As Task
                           Try
-                              RegisterLanguageService(GetType(IVbCompilerService), Function() Task.FromResult(_comAggregate))
+                              AddService(GetType(IVbCompilerService), Function(_1, cancellationToken, _2)
+                                                                          PreloadProjectSystemComponents()
```

Comment:
In the VB case, there's an explicit request for the service, so we can just add this without problems.